lib/jobs/install-os.js

```diff
@@ -310,7 +286,17 @@ function installOsJobFactory(
             self._subscribeNodeNotification(self.nodeId, function(data) {
                 assert.object(data);
 
-                return self.updateGraphProgress(self.options.progressMilestones.completed)
+                return Promise.resolve()
+                .tap(function() {
```

Comment:
Both `then` and `tap` are OK at here, but considering the event publish is just an action that "sneaked" into a normal procedure, so I'd prefer the `tap`